config/dev.yaml

```diff
@@ -245,6 +245,19 @@ fe:
     shell:
         - [/bin/false, Your administrator has not configured Grouper for shell management]
 
+    # A dict of lists of lists representing the options that users are allowed
```

Comment:
Yep can do. Do you think we should start deprecating the dedicated shell field? We could default to using it if it's present, otherwise it would use the general metadata_options field?